### ISSUE TITLE
Publish as npm package

### ISSUE THREAD

**Issue by royNiladri** [ansingh/casparser#1]
This is a very helful package and would love if i can get it from npm. Can you publish this?

**Comment by akash87** [comment]
@royNiladri did you test this? Does it still work?

**Comment by royNiladri** [comment]
Amazingly yes! I ran a detailed report and that was successful. Tweaked it a little to get current valuation as well.

**Comment by ansingh** [comment]
I would be happy to publish it. It's good to know it works for you - I extracted it out of my (incomplete) portfolio manager on someone's request on reddit and the only testing I had done is on mine and my family's portfolios.

I think it'll be useful to publish it as a library though, essentially what ImportHelper is. I still want to keep the main output JSON and as close to pdf as possible. Maybe with some additional utilities to export CSV, calculate XIRR, aggregate balance, etc. How does that sound ?

**Comment by royNiladri** [comment]
Yes I'd love to see the XIRR, the total invested vs total value now and such things. Ypu can start with a basics, whatever features you have and then keep adding more as and when you get time. Let me know if you need any help!🙂
